internal/cli/atlas/datalake/delete.go

```diff
@@ -36,7 +36,7 @@ func (opts *DeleteOpts) initStore() error {
 }
 
 func (opts *DeleteOpts) Run() error {
-	return opts.Delete(opts.store.DeleteDataLake, opts.ConfigProjectID(), opts.Entry)
+	return opts.Delete(opts.store.DeleteDataLake, opts.ConfigProjectID())
```

Comment:
drive by fix, the entry is handled internally and the meta programming of `opts.Delete` was saving us from our own errors